packages/react-front-kit/turbo/generators/config.ts

```diff
@@ -30,6 +30,12 @@ export default function generator(plop: PlopTypes.NodePlopAPI): void {
           templateFile: 'templates/test.hbs',
           type: 'add',
         });
+      } else {
```

Comment:
```suggestion
      }
      if (data?.type === 'pages') {
```
car il y a aussi le type layout donc le else ne convient pas